feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.directive.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Directive, Input, OnInit, OnDestroy } from '@angular/core';
+
+@Directive({
+  selector: '[cxConfiguratorMainAriaLabelledBy]',
+  standalone: true,
+})
+export class ConfiguratorMainAriaLabelledByDirective
+  implements OnInit, OnDestroy
+{
+  @Input('cxConfiguratorMainAriaLabelledBy') labelledbyId: string;
+
+  private mainEl: HTMLElement | null = null;
```

Comment:
please make protected (in order to ease extensibility)

Reply:
Changed as proposed.